modules/_labs/core/react/lib/CanvasProvider.tsx

```diff
@@ -16,8 +24,10 @@ export default class CanvasProvider extends React.Component<CanvasProviderProps>
     const {children, theme} = this.props;
 
     return (
-      <ThemeProvider value={theme}>
-        <InputProvider>{children}</InputProvider>
+      <ThemeProvider theme={theme}>
+        <DirectionContainer theme={theme} dir={theme.direction}>
```

Comment:
`dir` will set the attribute _and_ pass the `dir` prop through. You can drop `theme={theme}` and just do 

```tsx
const DirectionContainer = styled('bdo')<{dir: ContentDirection}>(
  ({dir}) => ({
    direction: dir,
  })
); 
```